src/utils/daos.c

```diff
@@ -630,6 +657,46 @@ common_op_parse_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 				D_GOTO(out_free, rc = RC_NO_HELP);
 			}
 			break;
+
```

Comment:
i think we should consider here the feedback that was given before.
instead of needing to always specify --src-pool ... --src-cont ... --dst_pool, etc..
maybe have 1 string instead:
--src={daos://pool/container or path} --dst={daos://pool/container or path}

what do you think? (maybe consult with others here like Bruno and Johann)

Reply:
@mchaarawi @johannlombardi @bfaccini I do agree having one string would simplify things. Is this something you want to handle in this PR? I ask because I imagine this is going to require adding some string parsing functions.

Reply:
well, I tend to agree with Mohamad, that would simplify a bit the command syntax and also have it look more like copy-commands common syntax. But this could also be part of a follow-on patch.

Reply:
@mchaarawi @daltonbohning @bfaccini The only concern I have about making this change in this PR is that then the data mover interface that is part of mpifileutils (dcp) will be different. Is it better if we leave it as is until we can update the other data mover interfaces?

Reply:
the problem if we don't make it is that people will start using this, and then we change it and they will have to change scripts / etc. similar to an API change, we should try to avoid unnecessary interface changes.

Reply:
@mchaarawi ok I will work on getting this added here